job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy

```diff
@@ -1036,6 +1036,7 @@ class PublisherContext extends AbstractExtensibleContext {
             tag rundeckContext.tag
             shouldWaitForRundeckJob rundeckContext.shouldWaitForRundeckJob
             shouldFailTheBuild rundeckContext.shouldFailTheBuild
+            shouldIncludeRundeckLogs rundeckContext.shouldIncludeRundeckLogs
```

Comment:
This element should only be included when version 3.4 or later of the rundeck plugin is installed